src/app/public/modules/media-query/media-query.service.ts

```diff
@@ -19,11 +19,30 @@ import {
 
 @Injectable()
 export class SkyMediaQueryService implements OnDestroy {
+
+  /**
+   * @internal
+   */
   public static xs = '(max-width: 767px)';
+
+  /**
+   * @internal
+   */
   public static sm = '(min-width: 768px) and (max-width: 991px)';
+
+  /**
+   * @internal
+   */
   public static md = '(min-width: 992px) and (max-width: 1199px)';
+
+  /**
+   * @internal
+   */
   public static lg = '(min-width: 1200px)';
 
+  /**
+   * @internal
```

Comment:
☝️ All of these properties are useful to our consumers; probably need to be documented.

Reply:
Documented.